### ISSUE TITLE
Plugin stopped working with new OKC layout

### ISSUE THREAD

**Issue by tfmorris** [benjaffe/chrome-okc-plugin#143]
I don't know if they've rolled out the new styling to everyone, but I got a new OKC style a week ago or so and the extension no longer works, presumably because they changed the names of various CSS classes.

Is anyone working an a fix for this?  I can take a look it if I get some free time, but someone who's familiar with the extension would probably make quicker progress.

**Comment by benjaffe** [comment]
I don't have the new styling yet... this happens every 6 months or so. Mrph.

@tfmorris Both of those hook points are added by the extension. Can you attach a screenshot, or describe what you're seeing (or not seeing)?

**Comment by tfmorris** [comment]
Basically nothing works.  This is more than just a little tweak.  It's a significant redesign with not only all new visuals, but all new CSS class names.  e.g. the match % class (old match-ratio-list?) is now userinfo2015-basics-asl-match  Overall, because it's a much more minimalistic design, there are many fewer anchor points to attach things to.  It'll probably take a little thought about how to redistribute the various information elements.

![screen shot 2015-10-21 at 5 55 50 pm](https://cloud.githubusercontent.com/assets/82178/10651431/83b4099c-781d-11e5-8250-acd81c56d1c4.png)

**Comment by benjaffe** [comment]
Thanks for the screenshot. I'll keep my eye out for the new layout.

**Comment by Wyrdbyrd** [comment]
The re-design went live on my and a few other people's profiles this past week.  There are comments about it in the Chrome Store Support section for your plug-in.

If I've already marked someone before the last OKC upgrade, I can at least see the watermarks on their profile picture. However, profiles are displayed differently now and the the plug-in isn't displaying Q&A data compatibility anymore.

**Comment by benjaffe** [comment]
It's been a month and I still don't have the redesign rolled out for my profile. :(

@mgeraci Is there a way your team can roll out the new design on a per-profile basis? I'd love to make the plugin work with OkC's new design, but I can't fix it until I see the design.

**Comment by mgeraci** [comment]
Sorry, Ben, that's not really possible. But don't give up hope; it should go up to everyone soon!

**Comment by mgeraci** [comment]
The new layout should be up to all desktop users now. Enjoy!

**Comment by benjaffe** [comment]
@mgeraci Thank you for the note!

@tfmorris @Wyrdbyrd I've pushed fixed for the worst issues. You'll start to see the plugin work again within the next day or so, or you can force an update from the Window - Extensions page (under the dev mode checkbox). I released it a few minutes ago, so it should be live within an hour. 

Right now, I know that hovering over the green/red bars looks ugly. Please document any other issues you notice.

**Comment by instantgreen** [comment]
Just checked it out and it even works with my 1280x1024 resolution. Thanks! :)

**Comment by tfmorris** [comment]
Thanks!  I'll have to take some screenshots so I remember where everything gets hooked in next time in case I want to tackle it myself.

The only nit I've noticed so far is that the detail flyout panel when hovering over the green/red bars gets rendered underneath both the original right column of the layout and the plugin's added right column.  It probably wants to be moved left a few pixels so it doesn't overlap your column and moved to the top of the render ordering so that it overlays everything else.

I just wish OKC hadn't "privileged" me with over a month of guinea pig status.

**Comment by benjaffe** [comment]
@tfmorris Haha, yeah, that'd annoy me too. I don't have enough energy to do this, but a refactor could allow the plugin to hook onto different things depending on what layout it sees. That wouldn't have helped this time though, since I didn't get the new layout until the end.

**Comment by tfmorris** [comment]
The latest changes fixed the flyout (since it's gone).  I'm on the fence about the whole accordion action for filtering the detail list, but I'll live with it for a bit to see if it grows on me.

There's something funky with the ellipsis continuation rendering on the detail list.  It seems that the question ellipsis (...) gets rendered when it's not needed and also there's possibly some crossover/confusion between the explanation ellipsis and the question ellipsis (or possibly the explanation ellipsis also, but independently, gets shown mistakenly when there's not explanation).

If I can spot a pattern, I'll add more detail and/or screenshots.

**Comment by benjaffe** [comment]
You're right on with the ellipsis. It always renders, since I'm not calculating whether it needs to. I should fix that.

I'm also seeing the accordion sometimes not working. I'll have to investigate that too.
